desktop/app-handlers/logging/ipc-handler/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
**Callout**: We use a handler in the main process to listen for log items transmitted from the renderer process.